### ISSUE TITLE
Error while loading mod (v1604.2.0.29, MC 1.16.3, Forge 34.1.42)

### ISSUE THREAD

**Issue by martymarty004** [FTBTeam/FTB-GUI-Library#10]
MC crashes while loading with the latest version, the log says that the mod requires a version of Forge that doesn't exist for 1.16.3
```
Error during pre-loading phase
net.minecraftforge.fml.ModLoadingException: Mod File ftb-gui-library-1604.2.0.29-forge.jar needs language provider javafml:35 or above to load
§7We have found 34.1
```

**Comment by MaxNeedsSnacks** [comment]
The new version of FTB GUI Library uses Architectury, which is a library that's unfortunately not available for versions below 1.16.4

**Comment by martymarty004** [comment]
I recreated my pack in 1.16.4, so this issue doesn't exist any more, but you should remove this version for 1.16.3
